packages/interface/src/pages/rounds/[pollId]/result/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { format } from "date-fns";
+import Image from "next/image";
+import { useMemo, useEffect } from "react";
+import { Hex, zeroAddress } from "viem";
+
+import { Heading } from "~/components/ui/Heading";
+import { useRound } from "~/contexts/Round";
+import { ResultItem } from "~/features/results/components/ResultItem";
+import { useProjectsResults } from "~/hooks/useResults";
+import { LayoutWithSidebar } from "~/layouts/DefaultLayout";
+import { useRoundState } from "~/utils/state";
+import { ERoundState, EMedal } from "~/utils/types";
+
+import type { GetServerSideProps } from "next";
+
+interface IResultPageProps {
+  pollId: string;
+}
+
+const ResultPage = ({ pollId }: IResultPageProps): JSX.Element => {
+  const { getRoundByPollId } = useRound();
+  const round = useMemo(() => getRoundByPollId(pollId), [pollId, getRoundByPollId]);
+
+  const projectsResults = useProjectsResults(
+    (round?.registryAddress ?? zeroAddress) as Hex,
+    (round?.tallyAddress ?? zeroAddress) as Hex,
+  );
+
+  const roundState = useRoundState({ pollId });
+
+  useEffect(() => {
+    if (projectsResults.data) {
+      return;
+    }
+
+    // eslint-disable-next-line no-console
+    projectsResults.refetch().catch(console.error);
+  }, [projectsResults, round]);
+
+  const decideMedal = (rank: number): EMedal | undefined => {
+    if (rank === 0) {
+      return EMedal.Gold;
+    }
+
+    if (rank === 1) {
+      return EMedal.Silver;
+    }
+
+    if (rank === 2) {
+      return EMedal.Bronze;
+    }
```

Comment:
I think it makes sense to use EMedal starting from 0. So you can use only one line:
```suggestion
    return rank <= EMedal.Bronze ? rank : undefined.
```